docs/rules/jsx-no-target-blank.md

```diff
@@ -18,6 +18,13 @@ There are two main options for the rule:
 * `{"enforceDynamicLinks": "always"}` enforces the rule if the href is a dynamic link (default)
 * `{"enforceDynamicLinks": "never"}` does not enforce the rule if the href is a dynamic link
 
+```json
+"react/jsx-no-target-blank": [<enabled>, { enforceDynamicLinks: <enforce> }]
+```
+
+* enabled: for enabling the rule. 0=off, 1=warn, 2=error. Defaults to 0.
```

Comment:
is this line necessary, since it's how every single eslint rule is configured?

Reply:
It's used on other pages (I copied it from one). For example: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/require-optimization.md